edx/edx-enterprise
# [ENT-996] Remove unique constraints on reporting config.

There seems to be no reason to actually have them. To quote myself:

> The Enterprise Customer used to be one-to-one to a reporting config, but later I switched it to be one-to-many instead because we should be able to have multiple data/report types per customer. However I thought there was a good reason to have a 1-to-1 relationship in the first place so I added the constraint on enterprise_customer to respect whatever reason that was. In hindsight there doesn't seem to be a good reason since we can still make multiple reporting configs per customer anyway...

**JIRA:** https://openedx.atlassian.net/browse/ENT-996

**Dependencies:** None.

**Merge deadline:** Whenever.

**Testing instructions:**

1. Run migrations with this in place.
2. Ensure you can add reporting configurations for the same customer as your heart desires.

**Merge checklist:**

  **DO NOT** just add dependencies to `requirements/*.txt` files.

**Post merge:**

**Author concerns:** List any concerns about this PR - inelegant 
solutions, hacks, quick-and-dirty implementations, concerns about 
migrations, etc.
